Configuration/TypoScript/Static/setup.typoscript

```diff
@@ -49,6 +49,7 @@ plugin.tx_blog {
             }
             posts {
                 maximumDisplayedItems = {$plugin.tx_blog.settings.lists.posts.maximumDisplayedItems}
+                numberOfLatestItems = {$plugin.tx_blog.settings.lists.posts.numberOfLatestItems}
```

Comment:
would more likely move this configuration into a dedicated upper section like `recent-/latestposts`